src/main/java/stsjorbsmod/actions/ExhumeAndMakeEtherealAction.java

```diff
@@ -0,0 +1,137 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.AbstractCard.CardType;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.cards.red.Exhume;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import com.megacrit.cardcrawl.powers.CorruptionPower;
+import stsjorbsmod.cards.cull.Seance;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class ExhumeAndMakeEtherealAction extends AbstractGameAction {
+    private AbstractPlayer p;
+    private static final UIStrings uiStrings;
+    public static final String[] TEXT;
+    private String prependDescription;
+
+    private ArrayList<AbstractCard> exhumes = new ArrayList<>();
+
+
+    public ExhumeAndMakeEtherealAction(String prependDescription) {
```

Comment:
I think it would make more sense for this class to contain the responsibility of setting the description prefix (using a UI string, see `IfEnemyIntendsToAttackAction` for example), rather than accepting it as a constructor param